docs/includes/schema-builder/galaxies_migration.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+use MongoDB\Laravel\Schema\Blueprint;
+
+return new class extends Migration
```

Comment:
If the migration class is not actually displayed in the docs, you can move code snippets in `up` and `down` methods into unit tests that would validate them (ensure the index is correctly created) and deleted.

Reply:
done!